static/docs/user-guide/install/plugins.md

```diff
@@ -0,0 +1,30 @@
+# IDE Plugins & Syntax Highlighting
+
```

Comment:
Also exist duplicated in https://dvc-org-pr-695.herokuapp.com/doc/user-guide/plugins. Let's merge?

Reply:
> Also exist duplicated in https://dvc-org-pr-695.herokuapp.com/doc/user-guide/plugins. Let's merge?

Next PR.

Reply:
Same as https://github.com/iterative/dvc.org/pull/695#discussion_r335282084

Reply:
Linke to #701 desc.